traffic_portal/test/integration/PageObjects/CDNPage.po.ts

```diff
@@ -139,18 +138,18 @@ export class CDNPage extends BasePage {
 
   public async DeleteCDN(cdn: DeleteCDN): Promise<boolean> {
     let name = cdn.Name + this.randomize;
-    let result = false;
     let basePage = new BasePage();
     await this.btnDelete.click();
     await this.txtConfirmName.sendKeys(name);
     await basePage.ClickDeletePermanently();
-    result = await basePage.GetOutputMessage().then(function (value) {
-      if (cdn.validationMessage == value) {
-        return true;
-      } else {
-        return false;
-      }
-    })
-    return result;
+    return await basePage.GetOutputMessage().then(value => cdn.validationMessage === value);
+  }
+  public async CheckCSV(cdn: CheckCSV): Promise<boolean> {
+    let linkName = cdn.Name;
+    if (await browser.isElementPresent(element(by.xpath("//span[text()='" + linkName + "']"))) == true) {
+      return true;
+    }else{
+      return false;
+    }
```

Comment:
This `else` is unnecessary; the `if` terminates with a `return` so all execution afterward is effectively `else`.

Reply:
Fix remove else.